api/utils.js

```diff
@@ -165,7 +165,8 @@ function createSearchArray (datasets) {
         const [prefixedSplitA, prefixedSplitB] = termToAdd.prefixed.split(':')
         // see https://github.com/zazuko/prefix-server/issues/26
         const iriSplitA = prefixedSplitB ? termToAdd.iri.value.split(prefixedSplitB)[0] : termToAdd.iri.value
-        Object.assign(termToAdd, { prefixedSplitA, prefixedSplitB, iriSplitA, iriSplitB: prefixedSplitB })
+        const ontologyTitle = prefixMetadata[prefixedSplitA].title || ''
```

Comment:
Possible at all the `prefixMetadata[prefixedSplitA]` will be `undefined`?

Reply:
Nope, here's why:
```js
const [prefixedSplitA, prefixedSplitB] = termToAdd.prefixed.split(':')
```

`prefixedSplitB` will be undefined for instance with `rdf:` , I don't `shrink` can generate something like: `:foo`